packages/components/popover/style/index.less

```
@@ -10,19 +10,23 @@
   background-color: @popover-background-color;
   border-radius: @popover-border-radius;
   box-shadow: @popover-box-shadow;
+  border: @popover-border-width solid @popover-border-color;
 
   &-wrapper {
     position: relative;
     word-wrap: break-word;
-    min-width: @popover-wrapper-min-width;
     padding: @popover-wrapper-padding;
     z-index: 1;
   }
 
-  &-title {
-    font-weight: @popover-title-font-weight;
-    min-height: @popover-title-min-height;
-    padding: @popover-title-padding;
+  &-header {
+    position: relative;
+    min-width: @popover-header-min-width;
+    min-height: @popover-header-min-height;
+    padding: @popover-header-padding;
+    font-weight: @popover-header-font-weight;
+    border-bottom: @popover-header-border-bottom;
+    z-index: 1;
```

```suggestion
    > .@{header-prefix} {
      padding: @drawer-header-padding;
```
这里应该只要改一下 padding 就好了吧。